fuse/direntry.go

```diff
@@ -30,6 +30,24 @@ func (d DirEntry) String() string {
 	return fmt.Sprintf("%o: %q", d.Mode, d.Name)
 }
 
+// DirEntryIno is a DirEntry struct extended by the "Ino" field for the
+// inode number.
+type DirEntryIno struct {
```

Comment:
Yes, that would make things much easier. But DirEntry is public API, and changing it may break users.